ci-operator/step-registry/ipi/conf/vsphere/osdisk/disktype/ipi-conf-vsphere-osdisk-disktype-commands.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH="${SHARED_DIR}/disktype.yaml.patch"
+
+if [ -n "${DISK_TYPE}" ]; then
+    cat > "${PATCH}" << EOF
+platform:
+  vsphere:
+    diskType: ${DISK_TYPE}
```

Comment:
This is default disk type, we also can set diskType for master and compute separately, right?
I suggest you can refer to `ci-operator/step-registry/ipi/conf/azure/osdisk/disktype/ipi-conf-azure-osdisk-disktype` to get them supported.

Reply:
i don't think so. because i not found any disktype field description for master and compute in vsphere installation doc.
but "compute.platform.azure.osDisk.diskType" & "controlPlane.platform.azure.osDisk.diskType" are in additional parameter list for in azure doc.
WDYT @jinyunma

Reply:
you are right, `openshift-install explain installconfig.compute.platform.vsphere` does not show custom diskType supported.